README.md

```diff
@@ -1,29 +1,39 @@
-# SLEIGH
+# SLEIGH Library
 
-SLEIGH was designed for the GHIDRA reverse engineering platform and is used to describe microprocessors with enough detail to facilitate two major components of GHIDRA, the disassembly and decompilation engines.
+[SLEIGH](https://ghidra.re/courses/languages/html/sleigh.html) is a language used to describe the semantics of instruction sets of general-purpose microprocessors, with enough detail to facilitate the reverse engineering of software compiled for these architectures. It is part of the [GHIDRA reverse engineering platform](https://github.com/NationalSecurityAgency/ghidra), and underpins two of its major components: its disassembly and decompilation engines.
 
-This repository provides a CMake build of SLEIGH so that it can be built and packaged as a standalone library.
+This repository provides a CMake-based build project for SLEIGH so that it can be built and packaged as a standalone library, and be reused in projects other than GHIDRA.
 
-## Dependencies
+## Supported Platforms
 
-| Name | Version |
+| Name | Support |
 | ---- | ------- |
-| [Git](https://git-scm.com/) | Latest |
-| [CMake](https://cmake.org/) | 3.21+ |
-| [Binutils](https://www.gnu.org/software/binutils/) | Latest |
-| [Zlib](https://zlib.net/) | Latest |
-| [Iberty](https://gcc.gnu.org/onlinedocs/libiberty/) | Latest |
-| [Doxygen](https://www.doxygen.nl/) | Latest |
-| [GraphViz](https://graphviz.org/) | Latest |
+| Linux | Yes |
+| macOS | Not yet |
```

Comment:
I haven't done much with it but it seems to build ok on MacOS too. Is this just in terms of "official" support?